public/manifest.webextension-shim.json

```diff
@@ -0,0 +1,4 @@
+{
+    "external-uri": ["^wc:", "^https?://arweave.net"],
```

Comment:
Is it RegExp? If so, what standard of RegExp is it?

Reply:
Perl Compatible?